monitor/service.go

```diff
@@ -210,6 +224,8 @@ func (m *Monitor) SetPointsWriter(pw PointsWriter) error {
 	m.mu.Unlock()
 
 	// Subsequent calls to an already open Monitor are just a no-op.
+
+	// FIXME: should we be running m.Open() here?
```

Comment:
`FIXME` or `TODO`?

Reply:
ya -- it seems like a bug to me that we're running m.Open() in this place.  it seems like something that needs to be examined with an eye for fixing it.